packages/cfd/src/Containers/cfd-change-password-confirmation.tsx

```diff
@@ -20,8 +20,9 @@ const ChangePasswordConfirmation = ({
         onSubmit={onConfirm}
     >
         {({ isSubmitting, handleSubmit }) => (
-            <form onSubmit={handleSubmit}>
+            <form onSubmit={handleSubmit} data-testid='cfd_change_password_form'>
```

Comment:
Suggestion based on our testing documentation.
```suggestion
            <form onSubmit={handleSubmit} data-testid='dt_cfd_change_password_form'>
```